acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/AcceptanceTestBase.java

```diff
@@ -47,6 +49,7 @@ protected AcceptanceTestBase() {
     transactions = new Transactions(accounts);
     web3 = new Web3(new Web3Transactions());
     pantheon = new PantheonNodeFactory();
+    contract = new ContractVerifier(accounts.getPrimaryBenefactor());
```

Comment:
NIT: naming this variable as `contract` can be misleading. Shouldn't we name it `contractVerifier`?

Reply:
Haha, you know that was it's name originally, I'll change it back